app/models/manageiq/providers/google/regions.rb

```diff
@@ -0,0 +1,50 @@
+module ManageIQ
+  module Providers::Google
+    module Regions
+      # From https://cloud.google.com/compute/docs/zones
+      REGIONS = {
```

Comment:
Yes, [there is](https://cloud.google.com/compute/docs/reference/latest/regions/list). Something to for me to update in one of the coming PRs?